nerfstudio/process_data/equirect_utils.py

```diff
@@ -33,17 +33,92 @@
 from nerfstudio.utils.rich_utils import ItersPerSecColumn
 
 
+def crop_bottom(bound_arr: list, fov: int, percent_crop: float) -> list:
```

Comment:
Add `_` to the start of these crop functions, I don't think we should expect others to use them.